sites/kit.svelte.dev/src/routes/+layout.server.js

```diff
@@ -0,0 +1,23 @@
+export const prerender = true;
+
+export const load = async ({ url, fetch }) => {
+	const nav_list = await fetch('/nav.json').then((r) => r.json());
+
+	return {
+		nav_title: get_nav_title(url),
+		nav_links: nav_list
+	};
+};
+
+/** @param {URL} url */
+function get_nav_title(url) {
+	const list = new Map([[/^docs/, 'Docs']]);
```

Comment:
This can just be a list of tuples instead of a map.

Reply:
svelte.dev uses a map like this, and that has multiple elements. Keeping a map here for consistency sake is not a bad idea IMHO